docs/src/modules/ROOT/pages/using-timefold-solver/modeling-planning-problems.adoc

```diff
@@ -1810,114 +1728,38 @@ To use a custom cloner, configure it on the planning solution:
 
 [source,java,options="nowrap"]
 ----
-@PlanningSolution(solutionCloner = NQueensSolutionCloner.class)
-public class NQueens {
+@PlanningSolution(solutionCloner = TimetableSolutionCloner.class)
+public class Timetable {
     ...
 }
 ----
 
-For example, a `NQueens` planning clone only deep clones all `Queen` instances.
-So when the original solution changes (later on during planning) and one or more ``Queen`` instances change,
+For example, a `Timetable` planning clone only deep clones all `Timetable` instances.
+So when the original solution changes (later on during planning) and one or more ``Timetable`` instances change,
 the planning clone isn't affected.
 
-[source,java,options="nowrap"]
-----
-public class NQueensSolutionCloner implements SolutionCloner<NQueens> {
-
-    @Override
-    public NQueens cloneSolution(CloneLedger ledger, NQueens original) {
-        NQueens clone = new NQueens();
-        ledger.registerClone(original, clone);
-        clone.setId(original.getId());
-        clone.setN(original.getN());
-        clone.setColumnList(original.getColumnList());
-        clone.setRowList(original.getRowList());
-        List<Queen> queenList = original.getQueenList();
-        List<Queen> clonedQueenList = new ArrayList<Queen>(queenList.size());
-        for (Queen originalQueen : queenList) {
-            Queen cloneQueen = new Queen();
-            ledger.registerClone(originalQueen, cloneQueen);
-            cloneQueen.setId(originalQueen.getId());
-            cloneQueen.setColumn(originalQueen.getColumn());
-            cloneQueen.setRow(originalQueen.getRow());
-            clonedQueenList.add(cloneQueen);
-        }
-        clone.setQueenList(clonedQueenList);
-        clone.setScore(original.getScore());
-        return clone;
-    }
-
-}
-----
-
 _The `cloneSolution()` method should only deep clone the planning entities._
```

Comment:
Let's add a partial code snippet for Timetable solution cloner.

Reply:
Custom cloners is one of those things that I'd rather not draw attention to. If people start asking how to do them, I'm going to convince them they shouldn't be doing them at all.